client/src/components/LinkTo.js

```diff
@@ -16,6 +16,15 @@ const MODEL_NAMES = Object.keys(Models).map(key => {
 	return camel
 })
 
+const modelPropTypes = MODEL_NAMES.reduce((map, name) => ({
+	...map,
+	[name]: PropTypes.oneOfType([
+		PropTypes.instanceOf(Models[name]),
+		PropTypes.object,
+		PropTypes.string,
+	])
+}), {})
+
```

Comment:
How did this change from `candidate-2.1` end up in this PR?

Reply:
Removed from PR

Reply:
Removed from PR